lldb/source/Breakpoint/Breakpoint.cpp

```diff
@@ -934,6 +943,12 @@ void Breakpoint::GetDescription(Stream *s, lldb::DescriptionLevel level,
     }
     s->IndentLess();
   }
+
+  // Reset the colors back to normal if they were previously greyed out.
+  if (s->AsRawOstream().colors_enabled())
```

Comment:
The comment says "if they were previously greyed out" but you're not actually checking that. I think with the helper variable the code is sufficiently self-explanatory that we can drop the comment. 
```suggestion
  // Reset the colors back to normal if they were previously greyed out.
  if (highlight_disabled)
```